packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.tsx

```diff
@@ -1,42 +1,38 @@
-import { languages, LanguageTag } from '@logto/language-kit';
+import { languages } from '@logto/language-kit';
 import resource, { isBuiltInLanguageTag } from '@logto/phrases-ui';
 import en from '@logto/phrases-ui/lib/locales/en';
 import { Translation } from '@logto/schemas';
-import cleanDeep from 'clean-deep';
-import { useEffect, useMemo } from 'react';
+import { useContext, useEffect, useMemo } from 'react';
 import { FormProvider, useForm } from 'react-hook-form';
 import { toast } from 'react-hot-toast';
 import { useTranslation } from 'react-i18next';
 import useSWR from 'swr';
 
 import Button from '@/components/Button';
-import useApi, { RequestError } from '@/hooks/use-api';
+import { RequestError } from '@/hooks/use-api';
 import Delete from '@/icons/Delete';
 
+import { CustomPhrasesContext } from '../../hooks/use-custom-phrases-context';
+import { CustomPhraseResponse } from '../../types';
```

Comment:
@/hooks @/types

Reply:
Hooks and types here are under the 'SignInExperience' folder.

<img width="348" alt="image" src="https://user-images.githubusercontent.com/10806653/193063447-87df8f7f-a2d5-4592-9a12-f2bec3b7c745.png">